applications/neuroscan/frontend/src/components/Sidebar/AddToViewerMenu.jsx

```diff
@@ -25,6 +25,33 @@ const AddToViewerMenu = ({
   const widgets = useSelector((state) => state.widgets);
   const viewers = getViewersFromWidgets(widgets);
 
+  function returnMenu() {
+    if (viewers.length > 0) {
+      return [
+        <Typography key="add-to-viewer-text">Add to existing viewer</Typography>,
+        viewers.map((viewer) => {
+          const instanceName = anchorEl?.parentNode?.textContent.replace('Add to', '');
+          let isEnabled = viewer.config.timePoint === timePoint;
+          if (isEnabled) {
+            viewer.config.instances.forEach((instance) => {
```

Comment:
please use `Array.prototype.findIndex()` if index != -1 then found and setEnabled = false

Reply:
@zsinnema fixed as requested